docs/source/contribute.rst

```diff
@@ -0,0 +1,33 @@
+Developer Information
+=====================
+
+How to contribute?
+------------------
+
+Contributions to this project are highly appreciated! You can either contact the maintainers 
+or directly create a pull request for your proposed changes:
+
+1. Fork the Project
+2. Create your Feature Branch (``git checkout -b feature/<descriptive-name>``)
+3. Commit your changes (``git commit -m 'Added NewFeature'``)
+4. Push to remote (``git push origin feature/<descriptive-name>``)
+5. Open a Pull Request to be merged with dev
+
+Developer information
+---------------------
+
+To contribute to this project, please follow these steps:
+
+1. Install `poetry <https://python-poetry.org/docs/>`_. We recommend installing ``poetry`` via `pipx <https://pypa.github.io/pipx/>`_ which gives you a global ``poetry`` command in an isolated virtual environment.
```

Comment:
We no longer recommend doing it via `pipx`, this has some problems I ran into and you should now always install a fresh poetry in the current venv, see README, maybe just copy over the latest steps from the most recent README